facturacion_mexico/facturacion_fiscal/timbrado_api.py

```
@@ -3241,106 +3376,277 @@ def _cascade_cancel_previous_after_substitute(new_ffm_name: str):
+			#   intento 1 = t0 · intento 2 = +0.5s · intento 3 = +1.0s
+			max_intentos = 3
+			last_error = None
+			for intento in range(1, max_intentos + 1):
+				try:
+					api = TimbradoAPI()
+					cancel_result = api.cancelar_factura(orig_si_name, "01", new_uuid)
+					frappe.logger().info(
+						f"Cancelación fiscal PAC (intento inmediato {intento}): {cancel_result}"
+					)
+					break
+				except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)